src/parser/types.ts

```diff
@@ -163,6 +170,10 @@ export interface GovernorConfigRaw extends GovernanceConfigDeprecatedRaw {
    * Quarry configuration
    */
   quarry?: QuarryRaw;
+  /**
+   * Token voter configuration.
+   */
+  ["token-voter"]?: TokenVoterRaw;
```

Comment:
```suggestion
  token-voter?: TokenVoterRaw;
```

Reply:
ah. That makes sense. Can we come up with a different name with no `-`?

Reply:
We could also go with lock or locker, I just though that might cause confusion.

Reply:
TokenLocker would probably be the most accurate because the address belongs to the locker contract.

Reply:
Actually, it's fine @MjCage. Let's just make `parse.ts` more consistent.